### PR TITLE
Feature/weather app

### PR DESCRIPTION
**Pull request by jpdf00**
The APP has a form field where the user can input their location and click on the button to search for weather information about the chosen location.
The user can choose the units for temperature.
The APP displays a GIF animation based on the current weather information. 
The APP is responsive.

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.scss"
```
Comment (krys2fa) [inline .github/workflows/linters.yml]: 💡 Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.
However, if you think that the change is necessary - open a [Pull Request in this repository](https://github.com/microverseinc/linters-config/blob/master/README.md#contributing).

**Review by krys2fa — CHANGES_REQUESTED** [review]
# Status: Marked as Invalid! ⚠️  
Please note that your Code Review Request is being marked as invalid. 
This is because the configs in .github/workflows/ must be exactly the same as that in the [Microverse linter config repo](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml). 




This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

**Comment by addod19** [comment]
## STATUS: INVALID
Hi @jpdf00, great job adding the recommended linter file for all Microverse students. However, I would mark the project as invalid because I cannot run any of the `npm` commands successfully, for example; `npm install` returns the error below

code1

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@6.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.hintrc
+      - name: Webhint Report
+        run: npx hint --telemetry=off .
```
Comment (rindrajosia) [inline .github/workflows/linters.yml:34]: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Status: Marked as Invalid! :warning:

Please note that your Code Review Request is being marked as invalid.
This is because the configs in .github/workflows/ must be exactly the same as that in the [Microverse linter](https://github.com/microverseinc/linters-config/blob/master/react-redux/.github/workflows/linters.yml) config repo.

This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

File: .github/workflows/linters.yml (hunk 2 above)
Comment (jpdf00) [reply .github/workflows/linters.yml:34]: I didn't.
The file is exactly the same as the one on the microverse linters repo.
I cloned the repo and copied the file.

I annexed a print from the micrverse repo and you tell me what is different
Screenshot_2021-04-29 microverseinc linters-config

Comment (jpdf00) [reply .github/workflows/linters.yml:34]: So Explain to me what is wrong again? Why did you waste my time invalidating a request claiming that I changed the linters configuration?

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work.  💻 👀 ✨
- Good work fixing the linter file.
- Excellent design.
- The code is well-structured and easy to understand.

## However, there are some changes required 👀 🧰 🔨 🔧
- The only detail I could find is that your `weather.js` file does not have error handling in case the API fails to respond to the request. Please add some error handling here.

***[OPTIONAL]***
- You may add current geolocation on page load so the user sees the current weather when they visit the page.

### Submit these changes and request a new code review! 😄 👍

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier)🧑‍💻🗨️🧑‍💻
# Happy Coding! 🧠 ⌨️ 🖱️ 🖥️

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @jpdf00 and @addod19  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**